MAINTAINERS.md

```diff
@@ -0,0 +1,10 @@
+# Maintainers
+| Maintainer       | GitHub ID                                             | Affiliation |
+|------------------|-------------------------------------------------------| ----------- |
+| Ian Hoang        | [IanHoang](https://github.com/IanHoang)               | Amazon |
+| Weifeng Hu       | [weifenh](https://github.com/weifenh)                 | Amazon |
```

Comment:
Should Weifeng be removed since he's left?

Reply:
Yes, I will remove and recommit